portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerItTest.scala

```diff
@@ -0,0 +1,169 @@
+          }
+        }
+      }
+
+    }
+
+    "DELETE project by id" should {
+      "return deactivated project entity" in {
+        val dummyUser: User = TestUserUtils.getDummyUser()
+        val userId = dummyUser.userId
+        val dummyProject = TestUserUtils.getDummyProject(userId)
+        val projectId = dummyProject.projectId
+        val accessToken = AccessTokenContent(userId.value)
+        val deactivatedProject = dummyProject.copy(active = false)
+
+        whenReady(userRepository.addUser(dummyUser)) { _ =>
+          whenReady(projectRepository.addProject(dummyProject)) { _ =>
+            Delete(s"/projects/${projectId.value}") ~> projectController.route(accessToken) ~> check {
+              status shouldBe StatusCodes.OK
+              responseAs[Project] shouldBe deactivatedProject
+            }
+          }
+        }
+      }
+
+      "return '404 Not Found' if there is no project with such id" in {
+        val dummyUser: User = TestUserUtils.getDummyUser(password = userPassword)
+        val userId = dummyUser.userId
+        val notExisitingProjectId = UUID.randomUUID().toString
+        val accessToken = AccessTokenContent(userId.value)
+
+        whenReady(userRepository.addUser(dummyUser)) { _ =>
+          Delete(s"/projects/$notExisitingProjectId") ~> projectController.route(accessToken) ~> check {
+            status shouldBe StatusCodes.NotFound
+          }
+        }
+      }
+
+      "return '403 Forbidden' if user is not an owner of that project" in {
+        val user: User = TestUserUtils.getDummyUser()
+        val anotherUser: User = TestUserUtils.getDummyUser()
+
+        val dummyProject = TestUserUtils.getDummyProject(anotherUser.userId)
+        val projectId = dummyProject.projectId
+        val accessToken = AccessTokenContent(user.userId.value)
+
+        whenReady(userRepository.addUser(user)) { _ =>
+          whenReady(userRepository.addUser(anotherUser)) { _ =>
+            whenReady(projectRepository.addProject(dummyProject)) { _ =>
+              Delete(s"/projects/${projectId.value}") ~> projectController.route(accessToken) ~> check {
+                status shouldBe StatusCodes.Forbidden
+                responseAs[String] shouldBe "Project deactivation is allowed for owners only"
+
+              }
+            }
+          }
+        }
+      }
+    }
+
+    "POST new project" should {
+      "return id of added project" in {
+        val dummyUser: User = TestUserUtils.getDummyUser(active = false)
+        val userId = dummyUser.userId
+        val dummyProject = TestUserUtils.getDummyProject(userId)
+        val projectAdditionRequest = ProjectAdditionRequest(
+          ownerId = userId,
+          name = dummyProject.name,
+          repository = dummyProject.repository
+        )
+        val accessToken = AccessTokenContent(userId.value)
+        val httpEntity = HttpEntity(`application/json`, Json.stringify(Json.toJson(projectAdditionRequest)))
```

Comment:
We don't use `HttpEntity`. 
It should be done by marshaller.

Reply:
fixed